Server/Server.csproj

```diff
@@ -59,6 +59,7 @@
     <WarningLevel>4</WarningLevel>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <Prefer32Bit>false</Prefer32Bit>
+    <LangVersion>7.1</LangVersion>
```

Comment:
does the language need to be 7.1? Unless there is specific syntax which requires 7.1 it should use the lowest possible. Otherwise you're forcing every coder to unnecessarily update

Reply:
I basically changed to this version so it would compile think it had problems compiling on default for some reason